projects/client/src/pages/Product/index.js

```diff
@@ -1,135 +1,84 @@
-      if (!props.login) {
-        toast.error("Please log in first", {
-          duration: 3000,
-        });
-      } else {
-        if (quantity === 1) {
-          let addCart = await axios.post(`http://localhost:8000/cart/addCart`, {
-            quantity,
-            price,
-            user_uid: uid,
-            product_id: id,
-          });
-          toast.success("Added to cart", {
-            duration: 3000,
-          });
-        } else {
-          if (quantity > stock) {
-            toast.error("Your cart has maximum stock of the product", {
-              duration: 3000,
-            });
-          } else {
-            let updateCart = await axios.patch(
-              `http://localhost:8000/cart/updateCart?user_uid=${uid}&product_id=${id}`,
-              {
-                quantity,
-                price,
-              }
-            );
-            toast.success("Added to cart", {
-              duration: 3000,
-            });
-          }
-        }
-        getCartFilterProduct();
-      }
-    } catch (error) {}
+  const handleQuantityChange = (type) => {
+    if (type === 'increase') {
+      setQuantity(quantity + 1);
+    } else if (type === 'decrease' && quantity > 1) {
+      setQuantity(quantity - 1);
+    }
   };
 
+  useEffect(() => {
+    getProductDetail()
+  }, [])
+
+  const { id } = useParams()
   return (
-    <>
-      <div> This is product page {id}</div>
-      <div> Stock: {stock}</div>
-      {stock > 0 ? (
-        <Button onClick={handleAddOrder}> This is product page {id}</Button>
-      ) : (
-        <Button isDisabled="true"> This is product page {id}</Button>
-      )}
-      <Toaster />
-    </>
-  );
+    <div>
+      <Card w='full'>
+        <Box borderWidth='1px' borderRadius='lg' overflow='hidden'>
+
+          <Image h='200px' w='full' p='2'
+            objectFit='contain' src={productData.image_url} />
+
+          <Text align='center'>
+            {productData.name}
+          </Text>
+          <Text align='center'>
+            {productData.price?.toLocaleString('id-ID', { style: 'currency', currency: 'IDR' })}
+          </Text>
+
+          <Box p='5'>
+            <Box w='full' as='button' mt={10}>
+              <ButtonGroup spacing='2'>
+                <IconButton
+                  aria-label='decrease quantity'
+                  icon={<MinusIcon />}
+                  onClick={() => handleQuantityChange('decrease')}
+                  size='sm'
+                  variant='outline'
+                  isDisabled={quantity === 1}
+                />
+                <Button w='full' bg={"#5D5FEF"} color='white' _hover={{ color: 'blue.500', bg: "white", border: '1px solid skyblue' }} >ADD TO CART ({quantity})</Button>
+                <IconButton
+                  aria-label='increase quantity'
+                  icon={<AddIcon />}
+                  onClick={() => handleQuantityChange('increase')}
+                  size='sm'
+                  variant='outline'
```

Comment:
add isDisabled as in decrement button. But for this one, set to disabled when quantity >= stock available